README.md

```diff
@@ -56,167 +59,125 @@ For updates on the latest meetup, please check us out on [Meetup.com](http://www
 
 ## Data SIM Card
 
-At Changi Airport (currency exchange counter) you could buy a $15 (5 days) / $30 (10 days) Singtel Prepaid SIM Card with up to 14 GB data.
-
-[![tourist_sim1](https://user-images.githubusercontent.com/761959/27039168-beac3314-4fbf-11e7-8ca8-704a92bc0921.png)](http://info.singtel.com/personal/phones-plans/mobile/prepaid/hitouristsimcards)
-[![tourist_sim2](https://user-images.githubusercontent.com/761959/27039169-beddf430-4fbf-11e7-93c6-89bd7d3a7de1.png)](http://info.singtel.com/personal/phones-plans/mobile/prepaid/hitouristsimcards)
+At Changi Airport you can buy a S$12 Singtel (100GB, 14 days) or Starhub (100GB, 10 days) plan. Alternatively, you can buy an Airalo eSIM before you arrive from US$4.50.
 
-You can go into most convenience stores (eg. 7-11) and buy a pre-paid simcard
+You can also go into most convenience stores (eg. 7-11) and buy a pre-paid simcard
 from either SingTel or StarHub (or possibly M1).
 
-[SingTel plans](http://info.singtel.com/personal/phones-plans/mobile/singtel-prepaid)
-and
-[StarHub plans](http://www.starhub.com/personal/mobile/prepaid-plans.html).
+- [SingTel plans](https://www.singtel.com/personal/products-services/mobile/prepaid-plans/hi-tourist)
+- [StarHub plans](https://www.starhub.com/personal/mobile/mobile-phones-plans/prepaid-cards/travel-sim.html).
+- [Airalo plans](https://www.airalo.com/singapore-esim)
+
+## Power Plugs
 
-Looks like [this is the card](https://www.facebook.com/singtel/posts/10152525153311677)
-to get for those who need nano sims.
+The standard in Singapore is 230V Type G (three pin) sockets.
+![Type G](https://www.iec.ch/themes/custom/iec/images/world-plugs/types/G/G_dia_sock_l.png)
 
-## Power (Socket)
+Most hotels have universal plugs that work with Type C (two pin) plugs, but you should consider bringing a travel adapter just in case.
 
-This [bad boy](http://www.iec.ch/worldplugs/typeG.htm) (Type G, BS 1363 UK, 230-240V).
+## Currency
+
+It's a good idea to bring [Singapore dollars
+(SGD)](https://en.wikipedia.org/wiki/Singapore_dollar) when visiting. However
+you can easily get cash out at ATMs after you arrive if necessary. While
+credit cards are widely accepted here, cash is still king, especially at Hawker centers. 
```

Comment:
I think providing a link to what "hawker centers" are will be helpful. 
https://en.wikipedia.org/wiki/Hawker_centre